packages/cli/src/generators/fmdapi.ts

```
@@ -0,0 +1,529 @@
+    const configToSearch = Array.isArray(parsed.config)
+      ? parsed.config
+      : [parsed.config];
+
+    targetDataSource = configToSearch.find(
+      (ds) =>
+        ds.path?.endsWith(dataSourceName) ||
+        ds.path?.endsWith(dataSourceName + "/") ||
+        ds.path === dataSourceName
+    );
+
+    if (targetDataSource && targetDataSource.layouts) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)